ci/test/06_script_a.sh

```diff
@@ -8,9 +8,7 @@ export LC_ALL=C.UTF-8
 
 BITCOIN_CONFIG_ALL="--disable-dependency-tracking --prefix=$DEPENDS_DIR/$HOST --bindir=$BASE_OUTDIR/bin --libdir=$BASE_OUTDIR/lib"
 DOCKER_EXEC "command -v ccache > /dev/null && ccache --zero-stats"
-if [ -z "$NO_DEPENDS" ]; then
-  DOCKER_EXEC ccache --max-size=$CCACHE_SIZE
-fi
+DOCKER_EXEC "command -v ccache > /dev/null && ccache --max-size=$CCACHE_SIZE"
```

Comment:
```suggestion
DOCKER_EXEC "ccache --max-size=$CCACHE_SIZE"
```

Ccache is a requirement, why would this not be there?

Reply:
> Ccache is a requirement for ci builds...

Yes, it is now. But it seems not guaranteed for custom defined `DOCKER_PACKAGES` macro:
https://github.com/bitcoin/bitcoin/blob/661e8df1b63b213d2d5b0d7cba0733869b508af9/ci/test/00_setup_env.sh#L56

Reply:
Maybe it is worth to disable ability to provide custom `DOCKER_PACKAGES` value as we have the `PACKAGES` macro on that purpose, no?

Reply:
I still like the ability to provide a custom DOCKER_PACKAGES. `ccache` is required for the ci in the same way that a compiler or python is required. Without it, it doesn't run.

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/18667#issuecomment-614706768).